### ISSUE TITLE
Plurimath LaTeX breaks in test document

### ISSUE THREAD

**Issue by opoudjis** [plurimath/plurimath#118]
https://github.com/metanorma/mn-samples-itu/

sources/STL-manual/en.adoc

Error:

````
Fatal Error: Expected one of [ITERATION EXPRESSION, ITERATION, dividend:ITERATION '\over' divisor:ITERATION EXPRESSION{0, }] at line 1 char 1.
bundler: failed to load command: metanorma (/Users/nickn/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/bin/metanorma)
/Users/nickn/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/parslet-2.0.0/lib/parslet/cause.rb:70:in `raise': Expected one of [ITERATION EXPRESSION, ITERATION, dividend:ITERATION '\\over' divisor:ITERATION EXPRESSION{0, }] at line 1 char 1. (Parslet::ParseFailed)
	from /Users/nickn/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/parslet-2.0.0/lib/parslet/atoms/base.rb:49:in `parse'
	from /Users/nickn/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/plurimath-0.2.3/lib/plurimath/latex/parser.rb:24:in `parse'
	from /Users/nickn/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/plurimath-0.2.3/lib/plurimath/latex.rb:12:in `to_formula'
	from /Users/nickn/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/plurimath-0.2.3/lib/plurimath/math.rb:43:in `parse'
	from /Users/nickn/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/metanorma-standoc-2.4.1/lib/metanorma/standoc/inline.rb:143:in `latex_parse1'
````

Candidate expressions

````
[latexmath%unnumbered]
++++
W(z)= { 1 - Q(z/\gamma_1) \over 1 - Q(z/\gamma_2) },
              0 <\gamma_2<\gamma_1\le1.
++++

[latexmath%unnumbered]
++++
    H_I^p(z) = K + \sum_{l=1}^{L}
                    { b_{0l}+b_{1l} z^{-1} \over
                      \ 1+c_{1l} z^{-1}+ c_{2l} z^{-2} \
                    }
++++

and of the cascade form (see <<IIR-cascade-filter>>), described
by the equation:

[latexmath%unnumbered]
++++
    H_I^c(z) = \prod_{l=1}^{N}
                    { b_{0l} + b_{1l} z^{-1} + b_{2l} z^{-2}\over
                           1 + c_{1l} z^{-1} + c_{2l} z^{-2}
                    }
++++
````

Plurimath Latex parsing is failing on one of these.

**Comment by ronaldtse** [comment]
@suleman-uzair this is urgent, can you please help? Thanks.

**Comment by ronaldtse** [comment]
@suleman-uzair, also:

Plurimath should NEVER raise any `Parslet::*` errors, but only Plurimath errors. Make it a `Plurimath::Formula::ParseFailure` error (or something like that).

**Comment by ronaldtse** [comment]
@suleman-uzair can you provide a replacement of that formula that works? Thanks.

**Comment by ronaldtse** [comment]
@suleman-uzair you can make a PR to to this file for now to fix that particular formula:

* Repo: https://github.com/metanorma/mn-samples-itu/
* File: sources/STL-manual/en.adoc

Then we add this formula into Plurimath and parse it correctly.

Thanks

**Comment by suleman-uzair** [comment]
@opoudjis  I have released a new version (0.2.4) for Plurimath with this fix.
